api/v1/database_types.go

```diff
@@ -113,17 +114,40 @@ type (
 		InitImage                string                        `json:"initImage,omitempty"`
 		ImagePullSecrets         []corev1.LocalObjectReference `json:"imagePullSecrets,omitempty"`
 		ImagePullPolicy          corev1.PullPolicy             `json:"imagePullPolicy,omitempty"`
+		Schedule                 []BackupSchedule              `json:"schedule,omitempty"`
 		ServiceAccountName       string                        `json:"serviceAccountName,omitempty"`
 		ContainerSecurityContext *corev1.SecurityContext       `json:"containerSecurityContext,omitempty"`
 		Resources                corev1.ResourceRequirements   `json:"resources,omitempty"`
 		Storages                 map[string]*BackupStorageSpec `json:"storages,omitempty"`
+		Annotations              map[string]string             `json:"annotations,omitempty"`
+		Labels                   map[string]string             `json:"labels,omitempty"`
+	}
+	BackupSchedule struct {
+		Name             string              `json:"name,omitempty"`
+		Enabled          bool                `json:"enabled,omitempty"`
+		Schedule         string              `json:"schedule,omitempty"`
+		Keep             int                 `json:"keep,omitempty"`
+		StorageName      string              `json:"storageName,omitempty"`
+		CompressionType  pbm.CompressionType `json:"compressionType,omitempty"`
+		CompressionLevel *int                `json:"compressionLevel,omitempty"`
+	}
+	BackupStorageProviderSpec struct {
+		// A container name is a valid DNS name that conforms to the Azure naming rules.
+		ContainerName string `json:"containerName,omitempty"`
+
+		Bucket            string `json:"bucket,omitempty"`
+		Prefix            string `json:"prefix,omitempty"`
+		CredentialsSecret string `json:"credentialsSecret"`
+		Region            string `json:"region,omitempty"`
+		EndpointURL       string `json:"endpointUrl,omitempty"`
+
+		// STANDARD, NEARLINE, COLDLINE, ARCHIVE for GCP
```

Comment:
Since you added this maybe we should also mention have the one for Azure
```suggestion
		// STANDARD, NEARLINE, COLDLINE, ARCHIVE for GCP
		// Hot (Frequently accessed or modified data), Cool (Infrequently accessed or modified data), Archive (Rarely accessed or modified data) for Azure
```